tests/Helpers.cs

```diff
@@ -26,4 +29,48 @@ internal static HttpContext SetClaimsPrincipal(this HttpContext httpContext, str
 
         return httpContext;
     }
+
+    internal static Borehole AddNewBoreholeToContext(this BdmsContext bdmsContext, Borehole boreholeToAdd)
```

Comment:
PP: Ich finde diese Helper-Methode ein bisschen einen Overkill, da wir prüfen, ob EF Core ein Borehole korrekt in die DB schreibt. Das sollte aus meiner Sicht stabil sein. 
Jedoch sehe ich den Punkt, dass man sich so ein paar Zeilen Code im Test spart. Kann man also stehenlassen, wenn du willst.

Reply:
Danke für das feedback. Finde diese Sichtweise gut. Habe zu stur den duplizierten code zu eliminieren versucht. Aber das ist nicht immer hilfreich. Finde die Lösung jetzt besser.